src/form/src/FormItem.tsx

```diff
@@ -166,6 +166,9 @@ export default defineComponent({
       if (feedback !== undefined && feedback !== null) return true
       return explainsRef.value.length
     })
+    const mergedDisabledRef = computed(() => {
+      return NForm?.disabled
+    })
```

Comment:
```suggestion
    const mergedDisabledRef = NForm ? toRef(NForm, 'disabled') : { value: false }
```

toRef 的开销会小一点，后半段的话可以试试行不行，类型不行的话可以把 useFormItem 里面接受的这个 mergedDisabled 从 ComputedRef 改成 Ref，如果还不行的话就写 ref(false)